airflow/providers/google/cloud/hooks/discovery_api.py

```diff
@@ -23,10 +23,10 @@
 
 from googleapiclient.discovery import Resource, build
 
-from airflow.providers.google.cloud.hooks.base import CloudBaseHook
+from airflow.providers.google.common.hooks.base import GoogleBaseHook
```

Comment:
Should we move this hook to common package?

Reply:
I think so, good catch! @potiuk WDYT?

Reply:
I will do it in a separate PR.

Reply:
Thanks for doing this.